src/qp/metrics/goodness_of_fit.py

```diff
@@ -0,0 +1,50 @@
+"""This module contains functions copied from the 1.10.0dev branch of Scipy.
```

Comment:
While it's well stated in the docstring and comments, these are unmodified copies of functions from Scipy 1.10.0dev. We should anticipate removing this code and using Scipy 1.10 when it is available. 

Lots of testing was done to ensure that the results from these functions match the results from the prior _non-vectorized_ implementation in Scipy 1.9.